alembic/config.py

```diff
@@ -285,7 +285,13 @@ def get_main_option(self, name, default=None):
         indicate a different section.
 
         """
-        return self.get_section_option(self.config_ini_section, name, default)
+        raw_section_option = self.get_section_option(
```

Comment:
this logic would take place in get_section_option(), since the "--name" option to alembic can make any section be used.   however I'm not sure this is the most appropriate place for the normpath to occur in any case.

There is already path normalization occurring in alembic/script/base.py such as in https://github.com/sqlalchemy/alembic/blob/master/alembic/script/base.py#L590 , so I dont think normalization is needed for version_locations, it needs to be in https://github.com/sqlalchemy/alembic/blob/master/alembic/script/base.py#L136 for script_location.